### PR TITLE
(CCD 1385) Added PIXAR headers to MIRI PHOTOM

### PR DESCRIPTION
**Pull request by hbrown-ST**
Missing keywords in delivered miri photom reference files did not trigger an error with crds certify. This change should fix that.

### CODE REVIEW

File: crds/jwst/specs/combined_specs.json
Comment (alphasentaurii) [inline crds/jwst/specs/combined_specs.json:1]: Isn't this supposed to be a MIRI PHOTOM tpn/spec modification?

Comment (hbrown-ST) [reply crds/jwst/specs/combined_specs.json:1]: Yeah it is, that was added after running install again. Weird.

**Review by alphasentaurii — APPROVED** [review]
LGTM pending confirmation that combined_specs.json are acceptable

File: crds/jwst/specs/combined_specs.json
Comment (hbrown-ST) [reply crds/jwst/specs/combined_specs.json:1]: Recreating the spec file didn't change a thing. I think this just didn't get picked up when I updated miri_gain.rmap for some reason in an earlier PR. Going to merge.

**Pull request merged**